types/lightpick/index.d.ts

```diff
@@ -262,32 +268,35 @@ declare namespace Lightpick {
         onClose?: Options.OnCloseFn;
 
         onError?: Options.OnErrorFn;
+
+        /**
+         * Triggered when the months select is changed.
+         */
+        onMonthsChange?: Options.OnMonthsChangeFn;
+
+        /**
+         * Triggered when the years select is changed.
+         */
+        onYearsChange?: Options.OnYearsChangeFn;
     }
 
     namespace Options {
         type Field = Element & { value: string };
 
-        enum DayOfWeek {
-            Monday = 1,
-            Tuesday,
-            Wednesday,
-            Thursday,
-            Friday,
-            Saturday,
-            Sunday,
-        }
+        type DayOfWeek = 1 | 2 | 3 | 4 | 5 | 6 | 7;
 
-        enum Orientation {
-            Auto = 'auto',
-            Left = 'left',
-            Right = 'right',
-            Top = 'top',
-            Bottom = 'bottom',
-            TopLeft = 'top left',
-            TopRight = 'top right',
-            BottomLeft = 'bottom left',
-            BottomRight = 'bottom right',
-        }
+        type Orientation =
+            | 'auto'
+            | 'left'
+            | 'right'
+            | 'top'
+            | 'bottom'
+            | 'top left'
+            | 'top right'
+            | 'bottom left'
+            | 'bottom right';
```

Comment:
After further testing I realised neither of the above enums is provided by the actual library.
Because of that I was hitting _TypeError: undefined is not an object_ errors at runtime.
Preferably I would leave it as `const enum` but that is disallowed by a dtslint rule.